src/main/java/com/github/webdriverextensions/junitrunner/WebDriverRunner.java

```diff
@@ -519,6 +520,13 @@ public BrowserConfiguration(Annotation annotation) {
                 desiredCapabilities = addCapabilities(desiredCapabilities, desiredCapabilitiesJsonMap);
             }
 
+            if( browserName.equals(BrowserType.CHROME) && System.getProperty(WebDriverProperties.CHROME_BINARY_PROPERTY_NAME) != null) {
```

Comment:
Yes, this seems better: I don't think it make sense to set the binary if its a remote driver.

I will pus a change